src/Faithlife.Tracing/ITrace.cs

```diff
@@ -2,10 +2,22 @@
 
 namespace Faithlife.Tracing
 {
+	/// <summary>
+	/// <see cref="ITrace"/> represents a span in a trace.
+	/// </summary>
+	/// <remarks>See <a href="https://github.com/opentracing/specification/blob/master/specification.md">OpenTracing Specification</a> for definitions of trace, span, etc.</remarks>
 	public interface ITrace : ITraceProvider, IDisposable
 	{
-		// TODO: recording an error?
+		/// <summary>
+		/// Sets a tag on this span.
+		/// </summary>
+		/// <param name="name">The tag name. This should be a value taken from <see cref="TraceTagNames"/>.</param>
```

Comment:
> This should be a value taken from TraceTagNames.

Reply:
Because it defines standard tag names. "Should", not "must".

Reply:
IMHO, "should" is overstating it. "Use values from `TraceTagNames` as appropriate. See also https://github.com/opentracing/specification/blob/master/semantic_conventions.md."